src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -807,5 +807,25 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
                 }
             }
         }
+
+#nullable enable
+        public sealed override MethodSymbol? PartialImplementationPart => _property is SourcePropertySymbol { IsPartialDefinition: true, OtherPartOfPartial: { } other }
+            ? (MethodKind == MethodKind.PropertyGet ? other.GetMethod : other.SetMethod)
+            : null;
+
+        public sealed override MethodSymbol? PartialDefinitionPart => _property is SourcePropertySymbol { IsPartialImplementation: true, OtherPartOfPartial: { } other }
+            ? (MethodKind == MethodKind.PropertyGet ? other.GetMethod : other.SetMethod)
+            : null;
+
+        internal bool IsPartialDefinition => _property is SourcePropertySymbol { IsPartialDefinition: true };
+        internal bool IsPartialImplementation => _property is SourcePropertySymbol { IsPartialImplementation: true };
+
+        public sealed override bool IsExtern
+        {
+            get
+            {
+                return PartialImplementationPart is { } implementation ? implementation.IsExtern : base.IsExtern;
```

Comment:
Extremely minor nit: you used arrow bodies for most of these, but not this one 😛.